program/src/instructions/update_whitelist_v2.rs

```diff
@@ -58,6 +58,7 @@ pub struct UpdateWhitelistV2<'info> {
     // New update authority, which must be a signer, if present, to prevent mistakes.
     pub new_update_authority: Option<Signer<'info>>,
 
+    // TODO: missing seeds?
```

Comment:
not needed; as long as update authority is a signer and the has one constraint ties it to the whitelist, it can update it